src/ttsim/plot/dag/tt.py

```diff
@@ -41,70 +46,105 @@ class NodeSelector:
 class _QNameNodeSelector:
     """Select nodes from the DAG."""
 
-    qnames: list[str]
+    qnames: set[str]
     type: Literal["neighbors", "descendants", "ancestors", "nodes"]
     order: int | None = None
 
 
 def tt(
-    policy_date_str: str,
+    # Args specific to TTSIM plotting
```

Comment:
```suggestion
    *,
    # Args specific to TTSIM plotting
```
Make the function keyword-only.